### DISCUSSION TITLE
How to change favicon.ico

### DISCUSSION THREAD

**Discussion by Datinhkhach** [Part-DB/Part-DB-server#644] [Q&A] [answered]
Hi
I want to change the favicon to my personal style. 

I overwrote the /public/favicon.ico file with the file I edited but it didn't work when reloading the site. Please give instructions on how to do it. Thanks a lot .

**Comment by jbtronics** [comment, marked as answer]
The favicon.ico is a very old format with not so good quality. Modern browsers will most likely use one of the icons in better resolution specified in the `public/icon/` folder (which file exactly will depend on the browser and purpose where it is shown).

**Comment by Datinhkhach** [comment]
Replaced public/icon/16x16. It's work , thanks you .
